code/modules/spells/spell_types/curse.dm

```diff
@@ -0,0 +1,29 @@
+/proc/curse_of_madness(mob/user, message)
+	if(user) //in this case either someone holding a spellbook or a badmin
+		to_chat(user, "<span class='warning'>You sent a curse of madness with the message \"[message]\"!</span>")
+		message_admins("[ADMIN_LOOKUPFLW(user)] sent a curse of madness with the message \"[message]\"!")
+		log_game("[key_name(user)] sent a curse of madness with the message \"[message]\"!")
+
+	for(var/mob/living/carbon/human/H in GLOB.player_list)
+		var/turf/T = get_turf(H)
+		if(T && !is_station_level(T.z))
+			continue
+		if(H.stat == DEAD)
+			continue
+		if(H.anti_magic_check(TRUE, FALSE))
+			continue
+		if(istype(H.get_item_by_slot(SLOT_HEAD), /obj/item/clothing/head/foilhat))
+			continue
+		H.playsound_local(H,'sound/hallucinations/veryfar_noise.ogg',40,1)
+		to_chat(H, "<span class='hypnophrase'>[message]</span>")
+		to_chat(H, "<span class='warning'>Your mind shatters!</span>")
```

Comment:
For clarties sake try putting something like "Your mind is shattered by the sudden revelation!"

Reply:
I think that the juxtaposition of saying something ridicolous followed by "Your mind shatters" might be more funny.

### **Hewwo!**
_Your mind shatters!_

Reply:
**Lavaland was a mistake!**
_Your mind shatters!_